api/apps/api/src/modules/projects/projects.controller.ts

```diff
@@ -206,27 +209,29 @@ export class ProjectsController {
     return;
   }
 
-  @ApiConsumesShapefile(true)
+  @ApiOkResponse({ type: PlanningAreaCreatedDto })
   @ApiOperation({
     description: 'Upload shapefile with project planning-area',
   })
+  @ApiConsumesShapefile(false)
```

Comment:
I had missed this earlier on - minor and can be for another PR if you think it's worth tweaking it: I suggest to use a config object for this decorator, such as `{ withGeoJsonResponse?: boolean }` defaulting to true, and maybe also adding the option to pass `type` as parameter (to avoid having a two ways of decorating this with `@ApiOkResponse()`.
But also because `@ApiConsumesShapefile(false)` is misleading until one looks at the source code of the decorator.
Again, this doesn't have much to do with the current PR but I'm just noting this because it seems a sensible and small tidy-up that can be done separately if you think it's worthy.

Reply:
ed1f455678144f55728ae940863249a58ca51cfa